tensorflow/core/BUILD

```diff
@@ -1092,6 +1096,7 @@ cc_library(
     deps = [
         ":protos_all_cc_impl",
         "//third_party/eigen3",
+        "@double_conversion//:double-conversion",
```

Comment:
It's a little unfortunate that mobile builds will end up using it, when I assume they don't need the extra speed of parsing (since they will only be parsing a few attribute values, not large files of text).  Do you know how large the double-conversion library is when compiled for android?

(one option if the net change in binary size would be too large is to have the double conversion library only used by the op that needs it for speed).

Reply:
It's not so large actually. The size of the compiled static library is 486KiB on my laptop.

Reply:
Stripped binary build with optimisation level 3 has size 77KiB. By the way, I can figure out what can we throw out from the library in the android build.